CMakeLists.txt

```diff
@@ -78,17 +82,17 @@ if(BUILD_EXAMPLES)
   endif()
 endif()
 
-if(BUILD_BENCHMARKS)
+if(GPUTREESHAP_BUILD_BENCHMARKS)
   include(${rapids-cmake-dir}/cpm/gbench.cmake)
   rapids_cpm_gbench()
 
   add_executable(BenchmarkGPUTreeShap benchmark/benchmark_gpu_treeshap.cu)
-  target_link_libraries(BenchmarkGPUTreeShap PRIVATE GPUTreeShap)
+  target_link_libraries(BenchmarkGPUTreeShap PRIVATE GPUTreeShap ${CMAKE_DL_LIBS})
```

Comment:
This fixes the problem observed in cuML CI, but this is a workaround and not a proper fix. I am unsure what the proper fix would be.